etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -99,18 +84,46 @@ def __init__(
             batch_size=self.batch_size,
         )
 
-        self._is_fitted: bool = False
+        self.__is_freezed: bool = False
+
+    @property
+    def is_freezed(self):
+        """Return whether to skip fit of a pretrained model."""
+        return self.__is_freezed
+
+    def freeze(self, is_freezed: bool = False):
+        """Change the state whether pretrained model can skip fit.
+
+        Parameters
+        ----------
+        is_freezed:
+            whether to skip fit of a pretrained model.
+        """
+        self.__is_freezed = is_freezed
```

Comment:
Is this ok that we could `freeze` untrained model?

Reply:
I think it is exactly not bad.
Otherwise it is needed to add one more flag to count number of fits that will look weird

Reply:
This question remains.